src/coreclr/jit/gcinfo.cpp

```diff
@@ -240,8 +240,8 @@ GCInfo::WriteBarrierForm GCInfo::gcIsWriteBarrierCandidate(GenTreeStoreInd* stor
     }
 
     // Ignore any assignments of NULL.
-    GenTree* const data = store->Data()->gtSkipReloadOrCopy();
-    if ((data->GetVN(VNK_Liberal) == ValueNumStore::VNForNull()) || data->IsIntegralConst(0))
+    GenTree* const data = store->Data()->gtSkipReloadOrCopy()->gtEffectiveVal();
```

Comment:
Why this change? `GT_COMMA` should not exist at this point, and if it did it is questionable to not skip reloads inside it (would cause issues during codegen).

Reply:
Agree, completely unnecessary, will remove as part of some other PR